.github/workflows/testAndPublish.yml

```diff
@@ -0,0 +1,314 @@
+        key: ${{ github.ref }}-${{ github.run_id }}
+
+  checkPot:
+    name: Check translator comments
+    runs-on: windows-latest
+    needs: buildNVDA
+    steps:
+    - name: Checkout cached build
+      uses: actions/cache/restore@v4
+      with:
+        path: ${{ github.workspace }}
+        key: ${{ github.ref }}-${{ github.run_id }}
+        fail-on-cache-miss: true
+    - name: Check comments for translators
+      run: ci/scripts/tests/translationCheck.ps1
+    - name: Upload artifact
+      if: ${{ success() || failure() }}
+      uses: actions/upload-artifact@v4
+      with:
+        name: makePot results
+        path: |
+          output/nvda.pot
+          output/potSourceFileList.txt
+
+  licenseCheck:
+    name: Check license compatibility of dependencies
+    runs-on: windows-latest
+    needs: buildNVDA
+    steps:
+    - name: Checkout cached build
+      uses: actions/cache/restore@v4
+      with:
+        path: ${{ github.workspace }}
+        key: ${{ github.ref }}-${{ github.run_id }}
+        fail-on-cache-miss: true
+    - name: License check
+      run: ci/scripts/tests/licenseCheck.ps1
+
+  unitTests:
+    name: Run unit tests
+    runs-on: windows-latest
+    needs: buildNVDA
+    steps:
+    - name: Checkout cached build
+      uses: actions/cache/restore@v4
+      with:
+        path: ${{ github.workspace }}
+        key: ${{ github.ref }}-${{ github.run_id }}
+        fail-on-cache-miss: true
+    - name: Run unit tests
+      run: ci/scripts/tests/unitTests.ps1
+    - name: Replace relative paths in unit test results
+      if: ${{ success() || failure() }}
+      # Junit reports fail on these
+      shell: bash
+      run: sed -i 's|file="..\\|file="|g' testOutput/unit/unitTests.xml
+    - name: Upload artifact
+      if: ${{ success() || failure() }}
+      uses: actions/upload-artifact@v4
+      with:
+        name: Unit tests results
+        path: testOutput/unit/unitTests.xml
+    - name: Publish unit test report
+      uses: mikepenz/action-junit-report@v5
+      if: ${{ success() || failure() }}
+      with:
+        check_name: Unit tests
+        detailed_summary: true
+        annotate_only: true
+        report_paths: testOutput/unit/unitTests.xml
+
+  createLauncher:
+    name: Create launcher
+    runs-on: windows-latest
+    needs: buildNVDA
+    steps:
+    - name: Checkout cached build
+      uses: actions/cache/restore@v4
+      with:
+        path: ${{ github.workspace }}
+        key: ${{ github.ref }}-${{ github.run_id }}
+        fail-on-cache-miss: true
+    - name: Set version variables
+      run: ci/scripts/setBuildVersionVars.ps1
+    - name: Set scons args
+      run: ci/scripts/setSconsArgs.ps1
+    - name: Prepare source code for launcher
+      shell: cmd
+      run: scons %sconsOutTargets% %sconsArgs% ${{ !runner.debug && '--all-cores' || '-j1'}}
+    - name: Build launcher
+      shell: cmd
+      run: scons launcher %sconsArgs% ${{ !runner.debug && '--all-cores' || '-j1'}}
```

Comment:
Looking into it again, I think this is obsolete since the step above already calls with %sconsOutTargets% and that contains launcher.

Reply:
Nice catch